src/components/navbar/help-dropdown.vue

```diff
@@ -0,0 +1,53 @@
+<!--
+Copyright 2020 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <li id="navbar-help-dropdown" class="dropdown">
+    <a class="dropdown-toggle" href="#" data-toggle="dropdown" role="button"
+      aria-haspopup="true" aria-expanded="false">
+      <span class="icon-question-circle"></span><span class="caret"></span>
+    </a>
+    <ul class="dropdown-menu">
+      <li>
+        <doc-link to="central-intro/">{{ $t('docs') }}</doc-link>
+      </li>
+      <li>
+        <a href="https://forum.getodk.org/" target="_blank">{{ $t('forum') }}</a>
+      </li>
+      <li>
+        <a href="/version.txt" target="_blank">{{ $t('version') }}</a>
```

Comment:
This will only work if using our Docker setup, right? It feels like Frontend shouldn't have that requirement. I wonder whether there maybe should be a pre-emptive query and the link is only shown if it's available?

Reply:
That's possible, but it's definitely more work. Do you have a use case in mind that doesn't use our Docker setup?

Reply:
Right. I guess we can make a note in the user documentation. It's true that we don't know of folks deploying frontend without the Docker setup.